packages/cicero-server/app.js

```diff
@@ -162,14 +164,122 @@ app.post('/draft/:template', async function (req, httpResponse, next) {
     }
 });
 
+/**
+ * Handle POST requests to /invoke/:template
+ * The body of the POST should contain the params, data and state.
+ * The clause is created using the template and the data.
+ * The call returns the output of requested clause.
+ *
+ * Template
+ * ----------
+ * The template parameter is the name of a directory under CICERO_DIR that contains
+ * the template to use.
+ *
+ * Request
+ * ----------
+ * The POST body contains six properties:
+ *  - sample or data
+ *  - parameters
+ *  - clause name
+ *  - state
+ *  - currentTime
+ *  - utcOffset
+ *
+ * Response
+ * ----------
+ * Output of the given clause from contract
+ *
+ */
+app.post('/invoke/:template', async function(req, httpResponse, next) {
+
+    try {
+        const options = req.body.options ? req.body.options : {};
+        const currentTime = req.body.currentTime ? req.body.currentTime : new Date().toISOString();
+        const utcOffset = req.body.utcOffset ? req.body.utcOffset : new Date().getTimezoneOffset();
```

Comment:
Consider Nullish Coalescing instead.
```suggestion
        const options = req.body.options ?? {};
        const currentTime = req.body.currentTime ?? new Date().toISOString();
        const utcOffset = req.body.utcOffset ?? new Date().getTimezoneOffset();
```

Reply:
Replaced with nullish coalescing at https://github.com/accordproject/cicero/pull/746/commits/8e56d2f5ea58d0babac1030613892216700da411. Also, upgraded ESlint version to support NC.